srcpkgs/git-secret/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'git-secret'
+pkgname=git-secret
+version=0.2.3
+revision=1
```

Comment:
can add noarch=yes, there are no ELF binaries, /usr/bin/git-secret is a bash script